src/openrct2/rct12/RCT12.cpp

```diff
@@ -199,7 +199,7 @@ uint8_t RCT12PathElement::GetRCT1SupportType() const
     return (flags & 0b01100000) >> 5;
 }
 
-uint8_t RCT12TrackElement::GetTrackType() const
+uint16_t RCT12TrackElement::GetTrackType() const
```

Comment:
this is still returning a `uint8_t` you're function that up converts it to a `uint16_t` is in the import code. You don't need to make this change.